app/src/main/java/com/bnyro/contacts/enums/SortOrder.kt

```diff
@@ -2,7 +2,8 @@ package com.bnyro.contacts.enums
 
 enum class SortOrder(val value: Int) {
     FIRSTNAME(0),
-    LASTNAME(1);
+    LASTNAME(1),
+    NICKNAME(2);
```

Comment:
#### FYI:
 No need to manually add integer `value` for enums. They have that  by default.
You can use `sortOrder.ordinal` instead of `sortOrder.value`

Reply:
Good to know, we should refactor that in the future (we're doing something similar in other You Apps too).